backend/src/routes/teams/management/rename.rs

```diff
@@ -11,7 +12,10 @@ use validator::Validate;
 
 #[derive(Debug, Serialize, Deserialize, ToSchema, Validate)]
 pub struct ChangeNameModel {
-    #[validate(length(min = 3, max = 32))]
+    #[validate(
+        length(min = 3, max = 32),
+        custom(function = "validate_username_chars")
```

Comment:
Because we are using it in multiple names `validate_username_chars` isn't very good name